pkgs/by-name/ca/caper/package.nix

```
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  lib,
+  ocaml,
+  ocamlPackages,
+  gnum4,
+  fetchFromGitLab,
+}:
+stdenv.mkDerivation rec {
+  pname = "caper";
+  version = "0.9";
+
+  src = fetchFromGitLab {
+    owner = "niksu";
+    repo = "caper";
+    rev = "v${version}";
+    hash = "sha256-TSryjz0NrGdkc+6vmfBqsuVpV3N9FvteTFsVqpUcm0w=";
+  };
+
+  nativeBuildInputs = [
+    ocaml
+    ocamlPackages.ocamlbuild
+  ];
+
+  buildInputs = [
+    ocamlPackages.findlib
+    ocamlPackages.menhir
+    ocamlPackages.angstrom
+    gnum4
+  ];
```

```suggestion
  nativeBuildInputs = [
    ocaml
    ocamlPackages.ocamlbuild
    ocamlPackages.findlib
    ocamlPackages.menhir
    gnum4
  ];

  buildInputs = [
    ocamlPackages.angstrom
  ];

  strictDeps = true;
```
After this change the PR can be merged in my opinion.
(the last line ensures build tools and actual dependencies are cleanly separated)